build/contacts.html

```diff
@@ -1,38 +1,62 @@
 <!DOCTYPE html>
 <html>
-<head>
+ <head>
  <title>Athlon Project</title>
  <meta charset="utf-8">
  <link rel="stylesheet" type="text/css" href="styles/style.css">
- <script src="https://www.google.com/recaptcha/api.js" async defer></script>
-<link href="style33.css?3f77ff0ef177a812c84b" rel="stylesheet"></head>
+ <link href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet">
+ <script src="https://code.jquery.com/jquery-3.3.1.min.js"></script>
+ <script type="text/javascript">
+    $(document).ready(function(){
```

Comment:
try writing this with vanilla javascript without  jquery !
Check
Dom ready:
http://youmightnotneedjquery.com/#ready
Click event:
http://youmightnotneedjquery.com/#on
Toggle class
http://youmightnotneedjquery.com/#toggle_class

Reply:
I'll have it done till midnight,